### DISCUSSION TITLE
Overlay for ImageCropper

### DISCUSSION THREAD

**Discussion by shatyuka** [CommunityToolkit/Windows#363] [Feedback]
Add an overlay to the inner crop area to show some UI placeholders.
https://github.com/CollapseLauncher/Collapse/issues/419

**Comment by shatyuka** [comment]
Wrote a simple implementation: https://github.com/shatyuka/CommunityToolkit.Windows/commit/7523277a37a4cce6aafd4db3346db32fcf7c7799

If this feature can be accepted, I will submit a PR.

**Comment by niels9001** [comment]
Hey @shatyuka, do you have a few screenshots to show what you want to achieve?

**Comment by shatyuka** [comment]
![310499043-72bf4605-751d-4810-80ab-59d28edf93f3](https://github.com/CommunityToolkit/Labs-Windows/assets/31368738/459a37c5-3f44-4d9f-a888-b86747e4d127)

![baee4f05707cec3c](https://github.com/CommunityToolkit/Labs-Windows/assets/31368738/dcbd0457-a17b-411e-b931-053757617a40)

**Comment by niels9001** [comment]
Thanks! Feel free to open a PR!

**Reply by michael-hawker** [reply]
Closing this discussion as PR was added here: https://github.com/CommunityToolkit/Windows/pull/364
